src/app/frontend/replicasetdetail/replicasetdetail.html

```diff
@@ -31,7 +31,8 @@
             <i class="material-icons kd-replicasetdetail-sidebar-icon">refresh</i>
             ROLLING UPDATE
           </span>
-          <span flex-offset="5" flex>
+          <span flex-offset="5" flex class="kd-replicasetdetail-sidebar-button"
```

Comment:
You can make it look like it looks now and be md-button: https://material.angularjs.org/latest/CSS/button

See flat buttons.